src/tests/notices/getNoticeDetails.js

```diff
@@ -0,0 +1,47 @@
+import { assert, statusOk } from "../../common/assertions.js";
+import {
+  noticeDetails,
+  NOTICES_API_NAMES,
+} from "../../api/notices.js";
+import defaultHandleSummaryBuilder from "../../common/handleSummaryBuilder.js";
+import { defaultApiOptionsBuilder } from "../../common/dynamicScenarios/defaultOptions.js";
+import { logErrorResult } from "../../common/dynamicScenarios/utils.js";
+import { getAuthToken,getNoticesList } from "../../common/utils.js";
+
+const application = "notices";
+const testName = "getNoticeDetails";
+
+// Dynamic scenarios' K6 configuration
+export const options = defaultApiOptionsBuilder(
+  application,
+  testName,
+  [
+    NOTICES_API_NAMES.getNoticeDetails,
+  ] // applying apiName tags to thresholds
+);
+
+// K6 summary configuration
+export const handleSummary = defaultHandleSummaryBuilder(application, testName);
+
+export function setup() {
+  return {
+    token: getAuthToken(),
+    noticesList: getNoticesList(getAuthToken())
+  };
+}
+
+export default (data) => {
+  for (const eventId of data.noticesList) {
```

Comment:
I won't make this as test, because this way inside the same test you are calling multiple times the same API...
instead each execution should perform a single test scenario.

You could choose between:
1. use same the first noticesList element
2. use a random list element
3. use a different element list for each invocation, restarting from the beginning of the list when the end of the list has been reached

Reply:
took 2 point as solution